Gulpfile.js

```diff
@@ -140,15 +177,14 @@ gulp.task('watch', function() {
   gulp.watch(['./src/js/**/*.js', 'Gulpfile.js'], ['js'], reload);
 });
 
-gulp.task('build', ['sass', 'images', 'fonts', 'js', 'templates']);
-gulp.task('build:optimized', ['sass:optimized', 'images:optimized', 'fonts', 'js', 'templates:optimized']);
+gulp.task('build', function (cb) {
+  runSequence('clean', ['sass', 'images', 'fonts', 'js', 'templates'], cb);
+});
 
-gulp.task('deploy', ['build:optimized'], function() {
```

Comment:
I would like the leave this, seeing as we use it on all of our projects.  Maybe instead just add a comment that indicates that this deploy task is Minimill specific?